pkg/framework/plugins/nodeutilization/usageclients.go

```diff
@@ -263,13 +274,13 @@ func (client *prometheusUsageClient) podUsage(pod *v1.Pod) (map[v1.ResourceName]
 	return nil, newNotSupportedError(prometheusUsageClientType)
 }
 
-func NodeUsageFromPrometheusMetrics(ctx context.Context, promClient promapi.Client, promQuery string) (map[string]map[v1.ResourceName]*resource.Quantity, error) {
+func NodeUsageFromPrometheusMetrics(ctx context.Context, logger klog.Logger, promClient promapi.Client, promQuery string) (map[string]map[v1.ResourceName]*resource.Quantity, error) {
```

Comment:
`NodeUsageFromPrometheusMetrics` is invoked within `sync` without any extra value. Why for the extra `logger`?